telegram/ext/_application.py

```diff
@@ -1133,7 +1133,7 @@ async def __create_task_callback(
                 return await asyncio.create_task(coroutine)
             # If user uses generator in python 3.12+, Exception will happen and we cannot do
             # anything about it. (hence the type ignore if mypy is run on python 3.12-)
-            return await coroutine  # type: ignore[misc]
+            return await coroutine
```

Comment:
This fails the pre-commit check - IIRC mypy wants to remove that when running on py3.1  🤔

Reply:
oh yeah. I even wrote a comment about it on the above line..